security/crowdsec/Makefile

```
@@ -0,0 +1,9 @@
+PLUGIN_NAME=		crowdsec
+PLUGIN_VERSION=		1.0.0
```

```suggestion
PLUGIN_VERSION=		0.0.1
PLUGIN_DEVEL=		yes
```

Try to start with a non 1.0.0 version as long as we give it a test as a development package

---

I'll go with 0.2